src/earn/EarnEnterAmount.tsx

```diff
@@ -59,8 +58,9 @@ const TOKEN_SELECTOR_BORDER_RADIUS = 100
 const MAX_BORDER_RADIUS = 96
 const FETCH_UPDATED_TRANSACTIONS_DEBOUNCE_TIME = 250
 
-function useTokens({ pool, mode }: { pool: EarnPosition; mode: EarnDepositMode }) {
-  const depositToken = useTokenInfo(pool.dataProps.depositTokenId)
+function useTokens({ pool }: { pool: EarnPosition }) {
+  const inputToken = useTokenInfo(pool.dataProps.depositTokenId)
```

Comment:
nit: confusing variable name calling this `inputToken` when it is returned as `depositTokens` just a few lines down. Should it just be called `depositToken`?

Reply:
Renamed in 1f1db1e072ce91ba1a3d84c8ae021edfdae23bdc!